### PR TITLE
fix: Top LPs are not sorted

### PR DESCRIPTION
**Pull request by BrodaNoel**
Fixing #123 

Please check that everything is OK. It's my first PR here

### CODE REVIEW

**Comment by BrodaNoel** [comment]
I just saw that Verified commit are required.
I just removed the previous commit and pushed a new one, but verified now.

File: src/queries/lpFills.ts [hunk 1]
```diff
@@ -117,4 +117,8 @@ export default async function getLpFills({
       alias: lp.alias || undefined,
     };
   });
+
+  lpFills.sort((a, b) => b.filledAmountValueUsd.comparedTo(a.filledAmountValueUsd));
```
Comment (niklasnatter) [inline src/queries/lpFills.ts:121]: now that we sort here, i think we can remove the sorting on line 74

**Review by niklasnatter — APPROVED** [review]
thanks a lot! would be great to remove the now obsolete `sort` call

**Comment by sukhoviianastasia** [comment]
Thanks for your work 😉

**Review by sukhoviianastasia — APPROVED** [review]

File: src/queries/lpFills.ts (hunk 1 above)
Comment (niklasnatter) [reply src/queries/lpFills.ts:121]: we will do this in a follow up to get the fix out - thanks again!

Comment (BrodaNoel) [reply src/queries/lpFills.ts:121]: @niklasnatter fixed in https://github.com/chainflip-io/chainflip-info-bot/pull/125

**Pull request merged**